src/main/java/seedu/address/logic/commands/contactlistcommands/ClearContactCommand.java

```diff
@@ -0,0 +1,35 @@
+package seedu.address.logic.commands.contactlistcommands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.logging.Logger;
+
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.model.ContactList;
+import seedu.address.model.Model;
+
+/**
+ * Clears the contact list.
+ */
+public class ClearContactCommand extends Command {
+
+    public static final String COMMAND_WORD = "clearcontact";
+    public static final String MESSAGE_SUCCESS = "Contact list has been cleared!";
+
+    private final Logger logger = LogsCenter.getLogger(ClearContactCommand.class);
```

Comment:
maybe logger doesn't need to be implemented if its not going to be used?

Reply:
Yes, will remove it.